packages/react-storage/src/components/StorageBrowser/views/utils.ts

```diff
@@ -34,14 +34,3 @@ export const compareDates = (a?: Date, b?: Date): number => {
   if (b === undefined) return -1;
   return a.getTime() - b.getTime();
 };
-
-export function isLastPage(
-  currentPage: number,
-  totalItems: number,
-  pageSize: number
-): boolean {
-  // Use Math.ceil so we can round up. For example, if you have
-  // 4 results, and your page size is 3, the last page (Math.ceil(4/3)) will be 2.
-  // If you have 3 results and your page size is 4, the last page will be 1
-  return Math.ceil(totalItems / pageSize) === currentPage;
-}
```

Comment:
The use cases for this util are now handled by `highestPageVisited` which has been added to the usePaginate hook.

Reply:
This logic was pretty tricky to get right from what i remember, were there unit tests that we migrated?

Reply:
No, I don't believe there were unit tests for the `isLastPage` util